File: RSPAClient/Core/AtPar.POCOEntities/MT_ATPAR_CONFIGURATION_SECTION_DTLS.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_ATPAR_CONFIGURATION_SECTION_DTLS
    {
        public string TAB_ID { get; set; }
        public string TAB_NAME { get; set; }
        public string PARAMETER_ID { get; set; }
        public string FRIENDLY_NAME { get; set; }
        public string DESCRIPTION { get; set; }
        public string TYPE { get; set; }
        public string VALIDATION_RULES { get; set; }
        public string DEFAULT_VALUE { get; set; }
        public string PARAMETER_VALUE { get; set; }
        public string TOOL_TIP_INFO { get; set; }
        public string VALID_FOR_ERP { get; set; }
        public string DISPLAY_FLAG { get; set; }
        public Nullable<short> DISPLAY_ORDER { get; set; }
        public string NEW_VALIDATION_RULES { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/MT_ATPAR_CONFIGURATION_SECTION_DTLS.ts
import { ATPAR_VALIDATION_RULES } from '../entities/atpar_validation_rules';

export class MT_ATPAR_CONFIGURATION_SECTION_DTLS {
    public TAB_ID: string;
    public TAB_NAME: string;
    public PARAMETER_ID: string;
    public FRIENDLY_NAME: string;
    public DESCRIPTION: string;
    public TYPE: string;
    public VALIDATION_RULES: string;
    public DEFAULT_VALUE: any;
    public PARAMETER_VALUE: any;
    public TOOL_TIP_INFO: string;
    public VALID_FOR_ERP: string;
    public DISPLAY_FLAG: string;
    public DISPLAY_ORDER?: number;     
    public validationRules: ATPAR_VALIDATION_RULES[];
    public NEW_VALIDATION_RULES: string;
}